mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -358,35 +357,36 @@ class ExperimentView extends Component {
     metricRanges,
     selected) {
 
+    let columnKey = 0;
     const numParams = paramKeyList.length;
     const numMetrics = metricKeyList.length;
     const row = [
-      <td><input type="checkbox" checked={selected}
+      <td key={columnKey++}><input type="checkbox" checked={selected}
         onClick={() => onCheckbox(runInfo.run_uuid)}/></td>,
-      <td>
+      <td key={columnKey++}>
         <Link to={Routes.getRunPageRoute(runInfo.experiment_id, runInfo.run_uuid)}>
           {runInfo.start_time ? Utils.formatTimestamp(runInfo.start_time) : '(unknown)'}
         </Link>
       </td>,
-      <td>{Utils.formatUser(runInfo.user_id)}</td>,
-      <td>{Utils.renderSource(runInfo)}</td>,
-      <td>{Utils.renderVersion(runInfo)}</td>,
+      <td key={columnKey++}>{Utils.formatUser(runInfo.user_id)}</td>,
+      <td key={columnKey++}>{Utils.renderSource(runInfo)}</td>,
+      <td key={columnKey++}>{Utils.renderVersion(runInfo)}</td>,
```

Comment:
Did these actually need keys? I'm not sure they're needed unless we generate stuff with a loop in here.

Reply:
"Warning: Each child in an array or iterator should have a unique "key" prop."
Because `row` here is an array, react needs a key. 
I'll change it to some hardcoded string.